cmake/flash/CMakeLists.txt

```diff
@@ -1,93 +1,80 @@
 # SPDX-License-Identifier: Apache-2.0
 
+add_custom_target(flash_runner_property_target)
+
+function(write content)
```

Comment:
Could you please add some header describing this function, and how to use it.

Secondly, having a function available in CMake, that is simply called `write` is too generic.

Could you make it a bit more specific, for example: `flash_runner_yaml_content_append` (maybe a bit long, so feel free to suggest something shorter).